<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startBt">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>671</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QGroupBox" name="DeviceGrp">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>DeviceType</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="webcamRb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Webcam</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="httpRb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Http client</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rtspRb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rtsp streaming</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fileRb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Video file</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="algolCb">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MIL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BOOSTING</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MEDIANFLOW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TLD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KFC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Algorithm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEd">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>http://192.168.0.100/image1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBt">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>340</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Boundingbox</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bbheightSb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="bbwidthSb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="WidhtLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
